packages/app/src/cli/services/build/extension.test.ts

```diff
@@ -138,6 +141,45 @@ describe('buildFunctionExtension', () => {
     expect(releaseLock).toHaveBeenCalled()
   })
 
+  test('performs wasm-opt execution by default', async () => {
```

Comment:
Is this covering when the `wasm_opt` option is missing? I think it may be covering when its `true` -- same behaviour? I see the schema above sets a default.